conn.go

```diff
@@ -9,18 +9,36 @@ import (
 	"time"
 )
 
+//DefaultConnTimeout time in seconds to wait for connection to beanstalk server.
```

Comment:
For consistency, please start comments with `// `, not `//`.

Reply:
@JensRantil do you mean a space after the "//", I can't quite tell the difference from your post.

Reply:
Oh, yes. Looks like Markdown scrubbed my space...